src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -63,6 +63,9 @@ public Result getResult() {
     public boolean hasRows() {
         return ArrayUtils.isNotEmpty(rows);
     }
+    public boolean hasDocString() {
+        return doc_string!=null;
```

Comment:
ArrayUtils.isNotEmpty(doc_string)

so empty string is treated as no doc_string